inc/saineighbor.h

```diff
@@ -50,6 +50,7 @@ typedef enum _sai_neighbor_entry_attr_t
 
     /**
      * @brief Destination MAC address for the neighbor
+     * validonly SAI_NEIGHBOR_ENTRY_ATTR_IS_LOCAL == true
```

Comment:
if this is mandatory_on_create and validonly, then you can change validonly to @condition and add it to the attributes if SAI_NEIGHBOR_ENTRY_ATTR_IS_LOCAL  is not conditional

Reply:
Is the change below what you had in mind?  This is failing the metachecker:
ASSERT FAILED (on line 1309): SAI_NEIGHBOR_ENTRY_ATTR_DST_MAC_ADDRESS: conditional attribute must be create only

    /**                 
     * @brief Destination MAC address for the neighbor
     *
     * @type sai_mac_t        
     * @flags MANDATORY_ON_CREATE | CREATE_AND_SET
     * @condition SAI_NEIGHBOR_ENTRY_ATTR_IS_LOCAL == true
     */
    SAI_NEIGHBOR_ENTRY_ATTR_DST_MAC_ADDRESS = SAI_NEIGHBOR_ENTRY_ATTR_START,